pynest/nest/lib/hl_api_connections.py

```diff
@@ -143,11 +142,9 @@ def Connect(pre, post, conn_spec=None, syn_spec=None,
 
     Notes
     -----
-    It is possible to connect arrays of nonunique node IDs by
-    passing the arrays as `pre` and `post`, together with a `syn_spec` dictionary.
-    However this should only be done if you know what you're doing. This will
-    connect all nodes in `pre` to all nodes in `post` and apply the specified
-    synapse specifications.
+    It is possible to connect Numpy arrays of node IDs one-to-one by passing the arrays as `pre` and `post`,
```

Comment:
```suggestion
    It is possible to connect NumPy arrays of node IDs one-to-one by passing the arrays as `pre` and `post`,
```
Please search and replace throughout the codebase. There are many ocurences of this in this PR.